src/olympia/amo/tests/test_helpers.py

```diff
@@ -226,8 +242,8 @@ def test_urlparams_returns_safe_string():
     s = render('{{ "https://foo.com/"|urlparams(param="a%20b") }}', {})
     assert s == 'https://foo.com/?param=a+b'
 
-    s = render('{{ "https://foo.com/"|urlparams(param="%AAA") }}', {})
```

Comment:
This wasn't valid, and now that I'm using a `QueryDict` it wouldn't work properly.